### PR TITLE
unit testcase for IBMCloudPowerVSProvider

### PR DESCRIPTION
**Pull request by mkumatag**
UT for IBMCloudPowerVSProvider support

### CODE REVIEW

File: pkg/cmd/provisioning/ibmcloud/create_service_id_test.go [hunk 1]
```diff
@@ -159,6 +159,35 @@ func TestCreateSharedSecrets(t *testing.T) {
...
+				require.NoError(t, err, "Failed to create temp directory")
+
+				testCredentialsRequestPowerVS(t, "firstcredreq", "namespace1", "secretName1", tempDirName)
+
+				return tempDirName
+			},
+			verify: func(t *testing.T, targetDir string, manifestsDir string) {
+				files, err := ioutil.ReadDir(targetDir)
+				require.NoError(t, err, "unexpected error listing files in targetDir")
+				assert.Zero(t, countNonDirectoryFiles(files), "Should be no files in targetDir when no CredReqs to process")
```
Comment (joelddiaz) [inline pkg/cmd/provisioning/ibmcloud/create_service_id_test.go:184]: The comment about "no files when no CredReqs to process" seems incorrect for this test case, as there is precisely 1 CredentialsRequest being processed.

**Comment by mkumatag** [comment]
@joelddiaz fixed the review comment, ptal at the latest commit.

**Review by joelddiaz — COMMENTED** [review]
/lgtm

**Comment by joelddiaz** [comment]
just adding automated test cases for already committed code. no docs/px/qe effect here
/label docs-approved
/label qe-approved
/label px-approved

**Pull request merged**